feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.spec.ts

```diff
@@ -229,40 +234,77 @@ describe('ConfigurationOverviewFormComponent', () => {
     checkConfigurationOverviewObs('a---a', 'xy', '--uv', '---uv--uv');
   });
 
-  it('should know if a configuration OV has attributes', () => {
-    initialize();
-    expect(component.hasAttributes(configCreate)).toBe(true);
-  });
+  describe('hasAttributes', () => {
+    it('should know if a configuration OV has attributes', () => {
+      initialize();
+      expect(component.hasAttributes(configCreate)).toBe(true);
+    });
 
-  it('should detect that a configuration w/o groups has no attributes', () => {
-    initialize();
-    const configWOOverviewGroups: Configurator.Configuration = {
-      ...ConfiguratorTestUtils.createConfiguration(
-        configId,
-        ConfiguratorModelUtils.createInitialOwner()
-      ),
-      overview: {
-        configId: ConfigurationTestData.CONFIG_ID,
-        productCode: ConfigurationTestData.PRODUCT_CODE,
-      },
-    };
-    expect(component.hasAttributes(configWOOverviewGroups)).toBe(false);
+    it('should detect that a configuration w/o groups has no attributes', () => {
+      initialize();
+      const configWOOverviewGroups: Configurator.Configuration = {
+        ...ConfiguratorTestUtils.createConfiguration(
+          configId,
+          ConfiguratorModelUtils.createInitialOwner()
+        ),
+        overview: {
+          configId: ConfigurationTestData.CONFIG_ID,
+          productCode: ConfigurationTestData.PRODUCT_CODE,
+        },
+      };
+      expect(component.hasAttributes(configWOOverviewGroups)).toBe(false);
+    });
+
+    it('should detect that a configuration w/o groups that carry attributes does not provide OV attributes', () => {
+      initialize();
```

Comment:
does the name fit - it looks more like
should detect that a configuration **with** groups that carry **no** attributes does not provide OV attributes